app/js/components/manageApps/Addon.jsx

```diff
@@ -268,16 +330,18 @@ class Addon extends Component {
 
   render() {
     const {
-      app, 
-      isOpen, 
-      showMessageDetail, 
-      affectedModules, 
-      action, 
+      app,
+      isOpen,
+      isAdmin,
+      showMessageDetail,
+      affectedModules,
+      action,
       loadingComplete,
       messageBody,
+      enableDeleteAndStart,
       messageType,
       showMessage } = this.state;
-      
```

Comment:
Fix indentation, this can also be seen in other areas of your code.

Reply:
Some codebase fix were done and my PR was not merged then so when i pull rebased this changes auto fixed themselves.